frontend/build.rs

```diff
@@ -32,6 +46,8 @@ fn shell(command: &str) {
         .expect("Couldn't write to build log");
     file.write_all(&output.stderr)
         .expect("Couldn't write to build log");
+
+    assert!(output.status.success(), "yarn couldn't build frontend");
```

Comment:
Message is not good. This function can execute any command (not only `yarn`). Please, include the actual command name inside the message. Also, it would be good to include exit code to the message